python/ray/worker.py

```diff
@@ -1728,11 +1752,8 @@ def connect(info,
 
     # Create a Redis client.
     redis_ip_address, redis_port = info["redis_address"].split(":")
-    worker.redis_client = thread_safe_client(
-        redis.StrictRedis(
-            host=redis_ip_address,
-            port=int(redis_port),
-            password=redis_password))
+    worker.redis_client = redis.StrictRedis(
```

Comment:
https://github.com/andymccurdy/redis-py says that Redis clients can safely be shared between threads (however pubsub clients cannot).

Reply:
We should leave a comment here about this thread-safe behavior.

Reply:
Fixed.